### ISSUE TITLE
Use native window borders

### ISSUE THREAD

**Issue by jhasse** [haiwen/seafile-client#829]
As Seafile uses it own window borders the following features don't work (Linux, MATE):

* Right-lick title menu
* Drag window to the left/right screen edge to half-tile
* Double-click or drag to top to maximize
* Theming
* Maximize button
* Close, minimize buttons on the left side

All these would be fixed by using the native window borders. Is there any reason not to? The settings window already does.

**Comment by htsmi** [comment]
Agree. I'm not a fan of apps that try to override the window manager. As a workaround if you use KDE and KWin, you can create a custom window rule to force set "No titlebar and frame" to no for "seafile-applet seafile client" class.

**Comment by mrjovanovic** [comment]
Same usability issues on KDE, Plasma version 5.20.5. None of these works, and also likely break accessibility. For example, because it does not use native window borders, it is difficult to distinguish when the window is focused for people with low vision. 

I don't think it would break branding to enable native window borders.
